table/server/underdb/glue/src/main/java/alluxio/table/under/glue/GluePartition.java

```diff
@@ -45,4 +45,27 @@ public String getSpec() {
   public Layout getLayout() {
     return mLayout;
   }
+
+  @Override
+  public String toString() {
+    StringBuilder sb = new StringBuilder();
+    sb.append("{");
+    if (mLayout.getData().getValuesList() != null) {
```

Comment:
related comment to above, but the spec should be in this output?

Also, is the spec included in the other partition types?

Reply:
Changed.